api/OffscreenCanvasRenderingContext2D.json

```
@@ -9,9 +9,25 @@
           },
           "chrome_android": "mirror",
           "edge": "mirror",
-          "firefox": {
-            "version_added": false
-          },
+          "firefox": [
+            {
+              "version_added": "preview",
+              "partial_implementation": true,
+              "notes": "See <a href='https://bugzil.la/1746750'>bug 1746750</a>."
```

(I'm trying to understand the newly introduced `impl_url`)
Should this use `impl_url`?

---

This only really says "bug" or "commit" URL. Not what it should be. So probably.

But the implication is that this should be a tracking bug for the actual implementation. Would be nice to have clarification the intent. 

```suggestion
              "impl_url": "https://bugzil.la/1746750",
```

---

Yeah, this should probably use the new `impl_url` property rather than a note.  A note will still need to be present with the `partial_implementation` key, which would be a good opportunity to describe how exactly the feature is "partial implementation"!

---

@queengooborg I don't think this should be partial at all. Instead we should remove the `partial_implementation` and indicate the support  for each part of it in the appropriate methods. 

I'm waiting on response from developers for what is supported: https://bugzilla.mozilla.org/show_bug.cgi?id=1746750#c18

@bsmth Have you started looking at test code that would allow us to verify which parts of the offscreencanvas have implementations? I suspect that this will mirror the canvas2D, but we don't "know"